src/js/helpers.js

```diff
@@ -40,8 +45,8 @@ function commadecimal_to_decimal(input) {
 
 // convert a decimal (1.25) into commadecimal (1,25)
 function decimal_to_commadecimal(input) {
-  if (isCents(input)) { // a bit misleading
-    return input.toString().replace('.', ',');
+  if (/^\d+\.?\d*$/.test(input)) {
+    return input.toFixed(6).replace('.', ',');
```

Comment:
This will conflict with the other PRs

Reply:
Should be okay if PR #110 is pulled first